docs/community.md

```diff
@@ -1,5 +1,19 @@
-Want to reach the ORAS community and developers?
-We're very interested in feedback and contributions for other artifacts.
+Welcome to the ORAS Project community.
+
+- [Join us](https://slack.cncf.io/) at [CNCF Slack](https://cloud-native.slack.com) under the **#oras** channel
+- Join the weekly community meeting at 4:00 PM PST at [Zoom meeting room](https://zoom.us/j/8079328631?pwd=UzN0NUlmbmcvTTN5L0o5VUQ2YndyQT09) with ID 807 932 8631. See [meeting notes](https://hackmd.io/P-O6n222TcSMoJgHmTTduw?view) for more details
```

Comment:
can you maybe not post the zoom meeting here, but in slack?

Reply:
Ah, I am curious why not show the Zoom meeting on the website?

Reply:
Minimize zoom bombing